src/downloadAzureProject/setupProjectFolder.ts

```diff
@@ -20,15 +20,13 @@ import { getNameFromId } from '../utils/azure';
 import { requestUtils } from '../utils/requestUtils';
 import { getRequiredQueryParameter } from './handleUri';
 
-// call this if starting from portal side (unchanged definition)
+// call this if starting from portal side
```

Comment:
no need for this comment either. looking at it from VSCode perspective, the URL could be invoked from anywhere really. It just so happens that we are lighting up the scenario from the portal.

Reply:
Ok, I deleted it